### ISSUE TITLE
Cannot disable notification: "Failed to connect to local Ollama instance"

### ISSUE THREAD

**Issue by geroldmeisinger** [continuedev/continue#1837]
### Before submitting your bug report

- [X] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [X] I've seen the [troubleshooting guide](https://docs.continue.dev/troubleshooting) on the Continue Docs

### Relevant environment info

```Markdown
- OS: Debian 12
- Continue: 0.8.42
- IDE: VSCode 1.91.1
- Model: -
- config.json:
  
{
  "models": [
    {
      "title": "codestral:22b-v0.1-q8_0",
      "provider": "ollama",
      "model": "codestral:22b-v0.1-q8_0"
    },
    {
        "title": "codestral",
        "provider": "ollama",
        "model": "codestral"
      }
    ],
  "customCommands": [
    {
      "name": "test",
      "prompt": "{{{ input }}}\n\nWrite a comprehensive set of unit tests for the selected code. It should setup, run tests that check for correctness including important edge cases, and teardown. Ensure that the tests are complete and sophisticated. Give the tests just as chat output, don't edit any file.",
      "description": "Write unit tests for highlighted code"
    }
  ],
  "tabAutocompleteModel": {
    "title": "codestral",
    "provider": "ollama",
    "model": "codestral"
  },
  "allowAnonymousTelemetry": false
}
```

### Description

![Screenshot from 2024-07-26 11-14-20](https://github.com/user-attachments/assets/c5eb9444-bad5-49cc-9a58-14e2eeda745d)

I sometimes work in parallel with other local AI models which require all VRAM thus I cannot start ollama. This notification shows up constantly, I cannot disable it and it is annoying. I don't want to disable auto-complete because it should work once Ollama runs again.

Related issues:
https://github.com/continuedev/continue/issues/1427
https://github.com/continuedev/continue/issues/1202

### To reproduce

1. Don't start Ollama
2. Start VSCode

**Comment by ganevdev** [comment]
Yes, the same problem. And notifications come EVEN if you disable all notifications from vs code. It's very annoying.

**Comment by CarloDePieri** [comment]
I can confirm this is indeed an issue and I hope it can be fixed soon.
Other programs need the gpu, which means we may run ollama only when needed, which means we need to be able to turn that notification off.

**Comment by gaelgerard** [comment]
My system is not strong enough to support Ollama. It would be a great thing to add an option to disable the plugin to look for Ollama.
Great plugin though!

**Comment by geroldmeisinger** [comment]
@github-actions I hate you
